sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala

```diff
@@ -235,7 +238,7 @@ case class GpuReplicateRows(children: Seq[Expression]) extends GpuGenerator with
   }
 }
 
-abstract class GpuExplodeBase extends GpuUnevaluableUnaryExpression with GpuGenerator {
+abstract class GpuExplodeBase extends GpuUnevaluableUnaryExpression with GpuGenerator with Logging {
```

Comment:
I don't see anywhere the `Logging` method is being used?

Reply:
Good catch, fixed.